tests/e2e/e2e_test.go

```diff
@@ -923,4 +923,21 @@ var _ = Describe("updateE2e", func() {
 			EnvVarTest("wildfly-"+appTestName, "binary", "keyvaluekey1value1")
 		})
 	})
+
+	Context("odo login", func() {
+		It("should login with username and password", func() {
+			runCmd("oc logout")
+			runCmd("odo login -u developer -p developer")
```

Comment:
There are no `-u` and `-p` in `odo login`

Reply:
updated, thanks

Reply:
Good catch

Reply:
Resolved Great :+1: